docs/document/content/reference/faq/_index.cn.md

```diff
@@ -320,3 +320,39 @@ ShardingSphere 中很多功能实现类的加载方式是通过 [SPI](https://sh
 更多关于 alias 使用方法请参考 [Proxool官网](http://proxool.sourceforge.net/configure.html)。
 
 PS：sourceforge 网站需要翻墙访问。
+
+## 27. [分片] 设置分片策略和自增主键策略时，*props*属性设置未生效？
+
+回答：
+
+须要特别注意自定义的算法名称约束（仅允许`[a-z][0-9]`和`-`）
+
+原因如下:
+
+ShardingSphere底层通过Spring的Binder来绑定Properties实例，假如格式不规范会抛出`InvalidConfigurationPropertyNameException`异常，但是ShardingSphere内部消化了这个异常，
```

Comment:
Can this reason be optimized so that spring exceptions can be thrown? Then the FAQ only needs to inform users that spring boot 2 has naming restrictions.

Reply:
Thank for you remind,I will submit an English version of the document and try to fix this problem.